applications.json

```diff
@@ -1,5 +1,22 @@
 {
   "applications" : [
+    {
+      "short_description" : "LICEcap simple animated screen capture tool for Windows and OS X .",
+      "categories" : [
+        "Images"
```

Comment:
I'm not sure it will work with a capitalized category name.

Reply:
@serhii-londar  It only produce gif file.

Reply:
I'm talking about:
"categories" : [
        "Images"
]

Reply:
Can you change to 
"categories" : [
        "images"
]
?

Reply:
@serhii-londar I got it.